components/cli/cmd/cellery/setup_create_existing_cluster.go

```diff
@@ -19,25 +19,107 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+	"github.com/cellery-io/sdk/components/cli/pkg/runtime"
 )
 
 func newSetupCreateOnExistingClusterCommand() *cobra.Command {
 	var isCompleteSetup = false
 	var isPersistentVolume = false
 	var hasNfsStorage = false
 	var isLoadBalancerIngressMode = false
+	var nfs runtime.Nfs
+	var nfsServerIp = ""
+	var fileShare = ""
+	var dbHostName = ""
+	var dbUserName = ""
+	var dbPassword = ""
 	cmd := &cobra.Command{
 		Use:   "existing",
 		Short: "Create a Cellery runtime in existing cluster",
 		Args: func(cmd *cobra.Command, args []string) error {
+			if isPersistentVolume && hasNfsStorage {
```

Comment:
Same. Args is meant for validating arguments. You can use PreRunE for validating user input

Reply:
updated.